ursa_core/src/lib.rs

```diff
@@ -1 +1,25 @@
+//!
+
+#![deny(
+    unused_variables,
+    unused_imports,
+    unused_import_braces,
+    unused_parens,
+    unused_lifetimes,
+    unconditional_recursion,
+    unused_extern_crates,
+    trivial_casts,
+    trivial_numeric_casts
+)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+
+mod error;
+
+pub use serde;
+pub use serde_bare;
+pub use serde_cbor;
+pub use serde_json;
+
+/// The errors generated by ursa components
```

Comment:
I would prefer it be documented in `error.rs`. That way the person re-exporting the module doesn't also have to document it.

Reply:
The values are documented. You also document each sub module. Not sure what you mean by your comment. Its common to have a short comment per sub module.

Reply:
Suppose I decided to re-export the submodule from another place. I'd have to copy this `mod` declaration with its doc-comment. If instead `errors.rs` included ```//! Errors generated by Ursa components``` it would be less cumbersome. 

Here it doesn't make much difference, but in some situations it can save you a lot of hassle, and is less error prone.